### DISCUSSION TITLE
laravel接口重定向跳转公众号授权链接提示跨域

### DISCUSSION THREAD

**Discussion by FH8023** [w7corp/easywechat#2701] [Q&A] [answered]
![微信图片_20230519165713](https://github.com/w7corp/easywechat/assets/49626993/b58edada-8479-4e47-a5b7-30a7db0f37d6)
![微信图片_20230519165732](https://github.com/w7corp/easywechat/assets/49626993/b6d90c96-01df-43c6-a02a-0abcbab8a0ec)
![微信图片_20230519165747](https://github.com/w7corp/easywechat/assets/49626993/20700f72-a81d-4f50-acce-d52db8d97379)
![微信图片_20230519165807](https://github.com/w7corp/easywechat/assets/49626993/e62d17a6-0842-4a20-a3ef-1cb1de537213)
代码如上
接口直接重定向该授权链接在微信开发工具中提示跨域，（已在路由添加跨域中间件 但是无效）
但是链接直接拿出来在微信客户端打开可以直接得到授权后的回调的接口的结果，麻烦告知下是否哪里要做调整

**Comment by overtrue** [comment, marked as answer]
看起来是前端代码用 ajax 读取了 test 返回的结果

**Reply by FH8023** [reply]
前端这一块用的是axios请求的test接口，是否需要换$.cors这样的方式呢

**Reply by overtrue** [reply]
肯定不能用 ajax 请求啊，而是直接跳转: http://yourdomain.com/test
